packages/canary/src/icons/chaos-engineering-icon.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
let's remove width and height from all svgs

Reply:
Don't forget that we today found out that we cannot use nucleo icons in most cases because Pixel Point alters the base icons. Until a better approach can be found we need to export the relevant icons from Figma, which is why you see width and height set. This should not be a merge blocker right now IMO.